ipatests/test_ipaserver/test_ipap11helper.py

```diff
@@ -48,6 +48,8 @@
 
 @pytest.fixture(scope="module")
 def p11(request):
+    if not os.path.isfile(SOFTHSM2_UTIL):
```

Comment:
please make this change as a separate commit

Reply:
I dropped the two lines.